tests/test_clinvar_to_evidence_strings.py

```diff
@@ -84,21 +71,21 @@ def test_return_none(self):
 
 class SkipRecordTest(unittest.TestCase):
 
+    #clinvar_record, clinvar_record_measure, consequence_type, allele_origin,
+                # allowed_clinical_significance, report
+
     def setUp(self):
         self.clinvar_record = test_clinvar.get_test_record()
         report = clinvar_to_evidence_strings.Report()
+        consequence_type = CT.ConsequenceType("ENSG00000163646", "stop_gained")
         # skip_record(clinvarRecord, cellbase_record, allowed_clinical_significance, counters)
-        self.args = [self.clinvar_record, {"reference": "A", "alternate": "T"},
-                     ["not provided"], report.counters]
+        self.args = [self.clinvar_record, self.clinvar_record.measures[0], consequence_type,
+                     "germline", ["not provided"], report]
```

Comment:
Signature of the skip_record:

`def skip_record(clinvar_record, clinvar_record_measure, consequence_type, allele_origin,
                allowed_clinical_significance, report)`

"germline" is the argument for the allele_origin parameter, ["not provided"] is the argument for the allowed_clinical_significance argument.